### PR TITLE
Change Select component's styles

### PR DESCRIPTION
**Pull request by Tankenstein**
Currency options now show the flag in every device size and keyboard focused options have the same style as hovered elements, rather than active ones.

### CODE REVIEW

File: src/select/Select.spec.js [hunk 1]
```diff
@@ -298,18 +310,18 @@ describe('Select', () => {
...
-    expect(findNthListElement(3).hasClass('active')).toBe(false);
+    expect(findNthListElement(3).hasClass('tw-dropdown-item--focused')).toBe(false);
     doTimes(3, () => component.simulate('keyDown', fakeKeyDownEventForKey(KEY_CODES.DOWN)));
-    expect(findNthListElement(3).hasClass('active')).toBe(true); // skips header!
+    expect(findNthListElement(3).hasClass('tw-dropdown-item--focused')).toBe(true); // skips header!
 
     doTimes(4, () => component.simulate('keyDown', fakeKeyDownEventForKey(KEY_CODES.DOWN)));
-    expect(findNthListElement(3).hasClass('active')).toBe(false);
-    expect(findNthListElement(8).hasClass('active')).toBe(true); // skips header and separator again!
+    expect(findNthListElement(3).hasClass('tw-dropdown-item--focused')).toBe(false);
```
Comment (oliverviljamaa) [inline src/select/Select.spec.js]: Could we encapsulate this to not repeat it?
`expect(focusIsOnListElementN(3)).toBe(false);`

Comment (Tankenstein) [reply src/select/Select.spec.js]: Ya sure 👍

File: src/moneyInput/MoneyInput.js [hunk 2]
```diff
@@ -166,7 +166,7 @@ export default class MoneyInput extends Component {
               <Fragment>
                 <i className="tw-money-input__keyline" />
                 <i
-                  className={`currency-flag currency-flag-${selectedCurrency.currency.toLowerCase()} hidden-xs m-r-2`}
+                  className={`currency-flag currency-flag-${selectedCurrency.currency.toLowerCase()} m-r-2`}
```
Comment (oliverviljamaa) [inline src/moneyInput/MoneyInput.js]: Don't we still want to hide the selected currency flag?

Comment (oliverviljamaa) [reply src/moneyInput/MoneyInput.js]: Or is this something else?

Comment (Tankenstein) [reply src/moneyInput/MoneyInput.js]: Originally, yes. However, i took a look. The smallest size we need to deal with is the SE, on which this starts to get nasty around 10 mil, so i figured let's just always show it

Comment (Tankenstein) [reply src/moneyInput/MoneyInput.js]: Maybe the label gets nasty actually?

Comment (oliverviljamaa) [reply src/moneyInput/MoneyInput.js]: The label was the original reason, yeah, I'd keep the behaviour the same for now and keep hiding?

Comment (Tankenstein) [reply src/moneyInput/MoneyInput.js]: Ok 👍

File: src/select/option/Option.spec.js [hunk 3]
```diff
@@ -35,4 +35,12 @@ describe('option', () => {
     component.setProps({ currency: 'hustle' });
     expect(component.find('i.currency-flag.currency-flag-hustle').length).toBe(1);
   });
+
+  it('does not show currency icons on mobile if it is selected', () => {
```
Comment (oliverviljamaa) [inline src/select/option/Option.spec.js]: Wait, I'm confused here... Isn't this a dropdown option and we would never hide its flag?

Comment (Tankenstein) [reply src/select/option/Option.spec.js]: This is also used for the "button" internal view of the select component

Comment (Tankenstein) [reply src/select/option/Option.spec.js]: Check out the `renderButtonInternals` method on `Select`

Comment (oliverviljamaa) [reply src/select/option/Option.spec.js]: 😮

**Pull request merged**